cypress/integration/rendering/sankey.spec.ts

```diff
@@ -0,0 +1,144 @@
+import { imgSnapshotTest, renderGraph } from '../../helpers/util.js';
+
+describe('Sankey Diagram', () => {
+  it('should render a simple example', () => {
+    imgSnapshotTest(
+      `
+      sankey-beta
+      
+      sourceNode,targetNode,10
+      `,
+      {}
+    );
+  });
```

Comment:
@nirname can you add some more imgSnapshotTests in a different PR?

Reply:
Could you explain how this test works? What is tested there exactly? What is expected value?